TODO/write-clean-css-10-simple-steps-pt2.md

```diff
@@ -34,10 +40,36 @@ Avoid bloated CSS
 
 **No.10: Manage your technical debt**
 
+**No.1-5 在**[**第一部分**]()。
+
+**No.6: 整理你的文件结构**
+
+保持文件结构的简洁
+
+如何在单个文件中构建你的 CSS 代码
+
+**No.7: 让 CSS 和 JaceScript 代码分离**
+
+为什么混合的 CSS 和 JavaScript 是大忌？
+
+如何更聪明地使用 CSS 和 JavaScript？
+
+**No.8: 留心供应商的前缀和后备方案**
+
+避免臃肿的 CSS 代码
+
+**No.9: 让供应商更容易维护，自动执行**
+
+**N0.10: 管理你的技术债务**
+
 # No.6: Lay out your file structure
 
+# No.6: 整理你的文件结构
+
 We ended the first part on how to write clean CSS by talking about modular CSS. Before we move to another step we should briefly discuss file structure. Modular CSS is useful on its own. However, it can quickly become overwhelming. Chances are that you are used to using a single stylesheet for all your CSS code. Then, going from one CSS stylesheet to, let’s say, 10 or 20 is a big difference. Let me give you one example of file structure following ITCSS.
 
+我们在第一部分讨论了模块化的 CSS 代码，在继续下一个方法之前我们应当简洁地讨论一下文件结构。独立来看，模块化 CSS 代码非常有用，然而模块多了之后就会无法控制。很可能你习惯将所有的 CSS 代码放入单个样式表文件，那么比如说，从一个样式表文件到十个或者二十个文件那就有如天壤之别了。我们来看写下面这个例子，这是 ITCSS 中的文件结构：
+
```

Comment:
However, it can quickly become overwhelming. Chances are that you are used to using a single stylesheet for all your CSS code。
然而,它很快就会变得无法控制，导致这样的原因是你习惯性的将所有CSS代码放入单个样式表文件中